tests/baselines/reference/user/ts-toolbelt.log

```diff
@@ -1,5 +1,12 @@
 Exit Code: 2
 Standard output:
+../../../../node_modules/@types/mocha/index.d.ts(2642,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'beforeEach' must be of type 'Lifecycle', but here has type 'HookFunction'.
+../../../../node_modules/@types/mocha/index.d.ts(2660,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'afterEach' must be of type 'Lifecycle', but here has type 'HookFunction'.
+../../../../node_modules/@types/mocha/index.d.ts(2676,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'describe' must be of type 'Describe', but here has type 'SuiteFunction'.
+../../../../node_modules/@types/mocha/index.d.ts(2697,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'xdescribe' must be of type 'Describe', but here has type 'PendingSuiteFunction'.
+../../../../node_modules/@types/mocha/index.d.ts(2711,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'it' must be of type 'It', but here has type 'TestFunction'.
+../../../../node_modules/@types/mocha/index.d.ts(2725,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'test' must be of type 'It', but here has type 'TestFunction'.
+../../../../node_modules/@types/mocha/index.d.ts(2732,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'xit' must be of type 'It', but here has type 'PendingTestFunction'.
```

Comment:
This test needs a `type: []` in its tsconfig to avoid pulling in the top-level mocha types from TS itself.